lib/console/hotspot_stats/hotspot_stats.ex

```diff
@@ -13,4 +13,168 @@ defmodule Console.HotspotStats do
     |> HotspotStat.changeset(attrs)
     |> Repo.insert()
   end
+
+  def get_all_query_for_string_sort() do
+    """
+      SELECT
+        stats.hotspot_address,
+        stats.packet_count,
+        stats.device_count,
+        h.name,
+        h.status,
+        h.long_city,
+        h.short_country,
+        h.short_state,
+        h.lat,
+        h.lng
+      FROM (
+        SELECT
+          DISTINCT(hotspot_address),
+          COUNT(hotspot_address) AS packet_count,
+          COUNT(DISTINCT(device_id)) AS device_count
+        FROM hotspot_stats
+        WHERE organization_id = $1 and reported_at_epoch > $2
+        GROUP BY hotspot_address
+      ) stats
+      LEFT JOIN hotspots h ON stats.hotspot_address = h.address
+      LEFT JOIN organization_hotspots os ON stats.hotspot_address = os.hotspot_address
+      ORDER BY
+        CASE $4 WHEN 'asc' THEN
+          CASE $3
```

Comment:
has to split up the queries because you cant mix integer columns and string columns for sorting in this case statement